packages/client/src/runtime/RequestHandler.ts

```diff
@@ -213,7 +214,7 @@ export class RequestHandler {
       throw new PrismaClientKnownRequestError(message, {
         code: error.code,
         clientVersion: this.client._clientVersion,
-        meta: error.meta,
+        meta: { modelName, ...error.meta },
```

Comment:
`modelName` is actually optional. Some top-level operations like `$queryRaw` don't have it. Could you ensure that `modelName: undefined` does not get added to `meta` in this case?